tests/full_test_file_trigger

```diff
@@ -1,8 +1,11 @@
-build/
-deploy/
-group_vars/
-resources/
-tests/
+build/Dockerfile
+deploy/10_service_account.yaml
+deploy/20_role.yaml
+deploy/30_role_binding.yaml
+deploy/40_cluster_role_kubevirt.yaml
+group_vars/all.yml
+resources/operator.yaml
+tests/common.sh
```

Comment:
Were the directory-based paths broken here? This seems like a strange addition to this commit.